server/routes/courtRegister/courtMapper.ts

```diff
@@ -58,39 +58,39 @@ export function courtBuildingMapper(building: CourtBuilding): CourtBuildingDetai
   return {
     id: building.id,
     code: building.subCode,
-    name: building.buildingName,
+    name: building.buildingName as string,
     addressline1: building.street,
     addressline2: building.locality,
     addresstown: building.town,
     addresscounty: building.county,
     addresspostcode: building.postcode,
     addresscountry: building.country,
-    contacts: building.contacts.map(courtBuildingContactMapper),
+    contacts: building.contacts?.map(courtBuildingContactMapper) || [],
   }
 }
 
 export function courtBuildingContactMapper(contact: CourtBuildingContact): CourtBuildingContactDetail {
   return {
     id: contact.id,
     type: contact.type,
-    number: contact.detail,
+    number: contact.detail as string,
   }
 }
 
 export function courtsPageMapper(
   courtsPage: CourtsPage,
   allCourtsFilter: AllCourtsFilter
 ): { courts: CourtDetail[]; pageMetaData: PageMetaData } {
-  const courts = courtsPage.content.map((court: Court) => courtMapper(court))
+  const courts = courtsPage.content?.map((court: Court) => courtMapper(court)) || []
   const pageMetaData = toPageMetaData(
-    courtsPage.first,
-    courtsPage.last,
-    courtsPage.empty,
-    courtsPage.totalPages,
-    courtsPage.totalElements,
-    courtsPage.number + 1,
-    courtsPage.size,
-    courtsPage.numberOfElements,
+    courtsPage.first as boolean,
```

Comment:
swagger is wrong on all of this since it  is using some odd plugin to generate this.
Maybe we should use the `Required` utility to indicate the real type